site/docs/api/pipeline.md

```diff
@@ -53,6 +53,10 @@ a `SourceStreamStage` which offers just these methods:
   provides a function to the source that determines the timestamp of
   each event.
 
+- `withIngestionTimestamps()`
+  declares that the source will assign the time of ingestion as the
+  event timestamp
```

Comment:
missing a `.`. Should we also say, instead of event-time semantics, system-time semantics will be used?